src/cli/expand-patterns.js

```diff
@@ -76,10 +76,11 @@ async function* expandPatternsInternal(context) {
           input: pattern,
         });
       } else if (stat.isDirectory()) {
+        const relativePath = path.relative(cwd, absolutePath) || ".";
```

Comment:
If the goal of this line is only to remove trailing slashes, why not do `.replace(/[/\\]+$/, '')` here to express the intent more clearly?

Reply:
I think the middle one `foo//bar` was effected too.

Reply:
I'll confirm when I back to laptop, if it's true, I'll add comment.

Reply:
Tested, the tailing slash is the only problem, applied your suggestion.

Reply:
Your solution can't work `\` can be the dirname, see this log https://github.com/prettier/prettier/runs/2746762536#step:6:1251

This commit https://github.com/prettier/prettier/pull/11000/commits/da461964378c29df112f67d7f44a52f1147e8955 works, but feel better/safer to use `path.relative`,  added comments https://github.com/prettier/prettier/pull/11000/commits/b00987886d90edaf9d34b3e45b56fd56ab659ebd